files/defaults.yaml

```diff
@@ -130,6 +130,9 @@ zowe:
     logLevel: "info"
     # Set to "exit" if you'd like startup to exit if any component has an error in the configure stage, otherwise zwe will warn but continue.
     onComponentConfigureFail: "warn"
+    startupChecks:
+      bypassAll: false
```

Comment:
Should `bypassAll` be respected by all other validation scripts, like the certificate check in zlux-app-server?

Reply:
Yes I hope people respect that and, that one will probably go away in the future because I have a draft of using certificate-analyser.jar universally instead of the nodejs-bound app-server one.